<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Encoder</class>
 <widget class="QDialog" name="Encoder">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>171</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>211</width>
    <height>171</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>211</width>
    <height>171</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Encoder</string>
  </property>
  <widget class="QPushButton" name="DataButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Проверить файлы данных</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EncodeButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Зашифровать</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DecodeButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Расшифровать</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
